### PR TITLE
Updated everything to .NET Standard 2.0 / ASP.NET Core 2.0

### PR DESCRIPTION
**Pull request by filipw**
Fixes https://github.com/WebApiContrib/WebAPIContrib.Core/issues/108

I also removed the few projects that were not maintained or used for anything (not even published as nuget - the `Concurrency` projects). If someone is interested in them we can restore them in the future.

The `WebApiContrib.Core.Razor` package was not updated, because it contains only one piece of functionality, and that is built on the extensibility point that was removed in ASP.NET Core 2.0. That functionality (overriding the behavior of `addTagHelper`) is not possible to do anymore.

As soon as this is merged, we can push all the Nuget packages.

### CODE REVIEW

File: src/WebApiContrib.Core.TagHelpers.Markdown/WebApiContrib.Core.TagHelpers.Markdown.csproj [hunk 1]
```diff
@@ -25,14 +21,12 @@
...
+    <PackageReference Include="Microsoft.AspNetCore.Mvc.Core" Version="2.0.2" />
+    <PackageReference Include="Markdig" Version="0.14.9" />
+    <PackageReference Include="Microsoft.AspNetCore.Razor.Runtime" Version="2.0.1" />
   </ItemGroup>
 
-  <ItemGroup Condition=" '$(TargetFramework)' == 'net451' ">
-    <Reference Include="System" />
-    <Reference Include="Microsoft.CSharp" />
+  <ItemGroup>
+    <Folder Include="Properties\" />
```
Comment (khellang) [inline src/WebApiContrib.Core.TagHelpers.Markdown/WebApiContrib.Core.TagHelpers.Markdown.csproj]: Remove this?

File: src/WebApiContrib.Core.WebPages/WebApiContrib.Core.WebPages.csproj [hunk 2]
```diff
@@ -13,21 +13,16 @@
...
+    <PackageReference Include="Microsoft.AspNetCore.Mvc.Core" Version="2.0.2" />
+    <PackageReference Include="Microsoft.AspNetCore.Mvc.ViewFeatures" Version="2.0.2" />
+    <PackageReference Include="Microsoft.AspNetCore.Mvc.Razor" Version="2.0.2" />
   </ItemGroup>
 
-  <ItemGroup Condition=" '$(TargetFramework)' == 'net451' ">
-    <Reference Include="System" />
-    <Reference Include="Microsoft.CSharp" />
+  <ItemGroup>
+    <Folder Include="Properties\" />
```
Comment (khellang) [inline src/WebApiContrib.Core.WebPages/WebApiContrib.Core.WebPages.csproj]: Same.

File: src/WebApiContrib.Core/WebApiContrib.Core.csproj [hunk 3]
```diff
@@ -13,20 +13,15 @@
...
+    <PackageReference Include="Microsoft.AspNetCore.Hosting" Version="2.0.1" />
+    <PackageReference Include="Microsoft.AspNetCore.Mvc.Core" Version="2.0.2" />
+    <PackageReference Include="Microsoft.Extensions.Configuration.Binder" Version="2.0.0" />
   </ItemGroup>
 
-  <ItemGroup Condition=" '$(TargetFramework)' == 'net451' ">
-    <Reference Include="System" />
-    <Reference Include="Microsoft.CSharp" />
+  <ItemGroup>
+    <Folder Include="Properties\" />
```
Comment (khellang) [inline src/WebApiContrib.Core/WebApiContrib.Core.csproj]: You get the idea... 😝

**Comment by damienbod** [comment]
looks good to me, thanks!

**Pull request merged**